mkdocs.yml

```diff
@@ -24,6 +48,7 @@ theme:
   palette:
     primary: "green"
     accent: "light green"
+>>>>>>> 3f8923bbcd9f08e13e9607478dc52595f08b27fe
```

Comment:
pozostałości po rozwiązywaniu konfilktów

Reply:
ehh usuwałem to nie wiem o co chodzi